install_config/install/stand_alone_registry.adoc

```diff
@@ -125,35 +125,75 @@ ifdef::openshift-enterprise[]
 [[registry-quick-installation]]
 === Quick Installation for Stand-alone OpenShift Container Registry
 
-When using the quick installation method to install stand-alone OCR, start the
+. When using the quick installation method to install stand-alone OCR, start the
```

Comment:
I generally prefer to not start a section without a intro paragraph. Maybe:
=== Quick Installation for Stand-alone OpenShift Container Registry
When using the quick installation method to install stand-alone OCR, the installation questions will be largely the same as if you were installing a full {product-title} PaaS. 
. Start the interactive installation by running:
 ----
 $ atomic-openshift-installer install
 ----
. When you reach the following screen, be sure to choose `2` to install a new registry: